skimage/morphology/footprints.py

```diff
@@ -95,6 +98,107 @@ def footprint_from_sequence(footprints):
     return morphology.binary_dilation(imag, footprints)
 
 
+def footprint_rectangular(shape, *, dtype=np.uint8, decomposition=None):
+    """Generate a rectangular or hyper-rectangular footprint.
+
+    Generates, depending on the length and dimensions requested with `shape`,
+    a square, rectangle, cube, cuboid, or even higher-dimensional versions
+    of these shapes.
+
+    Parameters
+    ----------
+    shape : tuple[int, ...]
+        The length of the footprint in each dimension. The length of the
+        sequence determines the number of dimensions of the footprint.
```

Comment:
```suggestion
        The size of the footprint in each dimension. The length of `shape`
        determines the number of dimensions of the footprint.
```

Reply:
No big deal, I just find it 'visually' confusing to read/see "The length ..." and then again, next sentence, "The length ..." when really they don't refer to the same thing at all.